test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -212,7 +229,10 @@ namespace iroha {
           execute(buildCommand(TestTransactionBuilder()
                                    .addAssetQuantity(asset_id, "1.0")
                                    .creatorAccountId(account->accountId())));
-      CHECK_ERROR_CODE(cmd_result, 2);
+
+      std::vector<std::string> query_args{
+          account->accountId(), "1.0", asset_id, "1"};
```

Comment:
"1.0" is repeated a lot in test, maybe replace it with a named constant? Also check other tests for this issue